CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+##Contributing
```

Comment:
Please remove this file and instead have a "Contributing" section in README.md that links here:

https://github.com/basho/riak-dotnet-client/wiki/Contributing

Then, merge this content with that page. This is to not have duplicate documentation. Thanks!